ai2thor/tests/test_unity.py

```diff
@@ -1003,6 +1003,27 @@ def test_get_interactable_poses(controller):
     ), "GetInteractablePoses with large maxDistance is off!"
 
 
+@pytest.mark.parametrize("controller", [wsgi_controller, fifo_controller])
+def test_get_reachable_positions(controller):
+    controller.reset("FloorPlan28")
+
+    event = controller.step("GetReachablePositions")
+    assert len(event.metadata["actionReturn"]) == 116
```

Comment:
From experience with `GetInteractablePoses`, this is probably going to break soon based on slight changes to Unity and non-determinism issues. I would each of these something like:

```python
event = controller.step("GetReachablePositions")
positions = event.metadata["actionReturn"]
assert 100 < len(positions) < 125
```
just to give it a bit more breathing room.

Then for `directionsRelativeAgent=True`, also assert that it has fewer positions than when `directionsRelativeAgent=False`.

Reply:
👍 as you predicted, this non-determinism problem happened immediately (failed test with a small mismatch, `111!=112`). I've gone ahead and improved the tests so that there is a range.

Reply:
Why can't we make GetInteractablePoses deterministic? Everything is static at the start. What makes it non-deterministic?